btv_site/blueprints/api/api.py

```diff
@@ -111,6 +114,27 @@ def api_raribox_post():
     db.session.commit()
     return jsonify({"error": False})
 
+@api.route("/viewercount", methods=["GET"])
+def api_viewercount_get():
+    time_past = (datetime.datetime.now() - datetime.timedelta(seconds = 10)).strftime('%Y-%m-%d %H:%M:%S')
+    viewerquery = db.session.query(StreamViewer).filter(StreamViewer.timestamp > time_past).all()
+    return jsonify(viewercount=len(viewerquery))
```

Comment:
In theory it should be more efficient to do `filter(...).count()` instead of `filter(...).all()` followed by `len()`.  The former can be turned directly into a db `COUNT()` query, the latter probably can't.